google/cloud/aiplatform/models.py

```diff
@@ -2892,8 +2892,8 @@ def __init__(
         # Create ModelRegistry with the unversioned resource name
         self._registry = ModelRegistry(
             self.resource_name,
-            location=location,
-            project=project,
+            location=location or self.location,
```

Comment:
Thank you for the contribution! Please update the unit tests to test this change.